service_capacity_modeling/models/org/netflix/evcache.py

```diff
@@ -214,13 +216,20 @@ def reserve_memory(instance_mem_gib):
     if cluster.count > (max_regional_size // zones_per_region):
         return None
 
+    services = network_services("evcache", context, desires, copies_per_region)
     ec2_cost = zones_per_region * cluster.annual_cost
 
+    # Account for the clusters and replication costs
+    evcache_costs = {"evcache.zonal-clusters": ec2_cost}
+    for s in services:
+        evcache_costs[f"{s.service_type}"] = s.annual_cost
```

Comment:
cross region costs are not applicable by default. Might need to check on a flag before giving the estimates.

Reply:
:+1: Done